test/app/components/Settings/Scheduler/createModal.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+describe('Create Modal Test', () => {
+    let CodeMirror;
+    let CreateModal;
+    beforeAll(() => {
+        configure({ adapter: new Adapter() });
+        global.document.createRange = function() {
```

Comment:
In the tests we already have, this is done in the global context.
See:
https://github.com/jakedex/falcon-sql-client/blob/498420c042490870b6669819ddfb8a1047cd2627/test/app/components/Settings/Preview/chart-editor.test.jsx#L5-L11

Reply:
Cool. I moved it to the global context, but if I'm not mistaken, you all don't already have polyfills for `document.createRange` so I'll keep the one I wrote 👍